lib/Loader/GraphBuilder.php

```diff
@@ -82,4 +89,17 @@ private function namespace(Package $package, $taskName): string
     {
         return sprintf('%s%s%s', $package->name(), Node::NAMEPSPACE_SEPARATOR, $taskName);
     }
+
+    private function shouldPurgeWorkspace(Package $package, ?Prototype $prototype): bool
```

Comment:
Note that this does not work properly -- it is not possible to disable purging for a specific package if the prototype says to purge.